src/ConfigEditor/ConfigEditor.test.tsx

```diff
@@ -88,6 +88,21 @@ describe('ConfigEditor', () => {
     });
   });
 
+  it('should enable WithEvent when it is toggled on', async () => {
+    const onChange = jest.fn();
+    render(<ConfigEditor {...props} onOptionsChange={onChange} />);
+
+    const withEventField = screen.getByTestId('data-testid withEvent');
```

Comment:
since you have defined the selector outside:
```suggestion
    const withEventField = screen.getByTestId(selectors.components.ConfigEditor.WithEvent.testID);
```